sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
I think this was fine before, because with the current sentence I understand that requested date from the sale order will be passed to lines, which is not happening.